homeassistant/components/fronius/__init__.py

```diff
@@ -195,8 +254,19 @@ async def _get_inverter_infos(self) -> list[FroniusDeviceInfo]:
                     unique_id=unique_id,
                 )
             )
+            _LOGGER.debug(
+                "Inverter found at %s (Device ID: %s, UID: %s)",
+                self.host,
+                solar_net_id,
+                unique_id,
+            )
         return inverter_infos
 
+    @callback
+    def _rescan_inverter(self, now: datetime) -> None:
+        """Initiate a scheduled rescan of available inverters."""
+        self.hass.async_add_job(self._init_devices_inverter())
```

Comment:
This should not be wrapped in a HassJob. `async_track_time_interval` already does that, and also takes care of cancellation. The job-in-job would probably keep running.

Reply:
Okay. Very sensible. Adjusted accordingly